integration/env/azure.go

```diff
@@ -94,6 +102,16 @@ func init() {
 	}
 }
 
+func AzureAvailabilityZones() []int {
+	azs := strings.Split(strings.TrimSpace(azureAvailabilityZones), " ")
+	zones := make([]int, len(azs))
+
+	for i, s := range azs {
+		zones[i], _ = strconv.Atoi(s)
```

Comment:
We should never ever hide errors. When you think the error case is unlikely to ever happen for whatever reasons and you do not want to add the error to the interface then at least panic here. Next year something changes that nobody is aware of anymore and we waste a lot of time and man power because we were too lazy to invest 10 seconds to handle an obvious error case.

Reply:
Fixed, please take a look.